tests/test_lint.py

```diff
@@ -7,6 +7,12 @@
 import petab
 import pytest
 from petab import (lint, sbml)  # noqa: E402
+from petab.C import *
+
+# import fixtures
+pytest_plugins = [
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Constant name "pytest_plugins" doesn't conform to UPPER_CASE naming style](https://app.codacy.com/manual/dweindl/PEtab/pullRequest?prid=4859825)